dynamodb/client.go

```diff
@@ -0,0 +1,95 @@
+package dynamodbclient
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+)
+
+const (
+	TableHashKey = "conceptId"
+)
+
+type Model struct {
+	UUID         string   `json:"conceptId"`
+	ConcordedIds []string `json:"concordedIds"`
+}
+
+type DynamoDbClient interface {
```

Comment:
I would have `type Client interface` instead (or `Clienter` if you wanted to stick to Go idioms, but it doesn't quite sound right)